packages/trader/src/AppV2/Components/RiskManagementItem/risk-management-item.tsx

```diff
@@ -150,10 +150,9 @@ const RiskManagementItem = observer(
                                     textAlignment='center'
                                     status={errorMessage ? 'error' : 'neutral'}
                                     name={type}
-                                    unit={currency}
+                                    unitRight={currency}
```

Comment:
After I updated quill ui library to the latest version, this place became a source of errors as props, which this component accepts, were changed.  So I just replace old one with new, screenshot of result is pinned below
![Screenshot 2024-06-26 at 9 07 06 AM](https://github.com/binary-com/deriv-app/assets/121025168/1896d0be-7774-41d0-b6e5-571631a726bc)